readme sulution.md

```diff
@@ -0,0 +1,2 @@
+[DEMO LINK](https://kostya-flern.github.io/layout_human-rights/) <br>
```

Comment:
no need to create new readme file, you already have it in repo. You need to update it.